utils/cloud_utils.ts

```diff
@@ -20,3 +21,17 @@ export function isCustomerCardExpired(customer?: CloudCustomer): boolean {
     const lastExpiryDate = new Date(expiryYear, customer.payment_method.exp_month, 1);
     return lastExpiryDate <= new Date();
 }
+
+export function billingDetailsIsValid(billingDetails: BillingDetails): boolean {
+    if (billingDetails == null) {
+        return false;
+    }
+    return Boolean(
+        billingDetails?.address &&
+        billingDetails?.city &&
+        billingDetails?.state &&
+        billingDetails?.country &&
+        billingDetails?.postalCode &&
+        billingDetails.name,
+    );
+}
```

Comment:
I believe we already have a method for this here: https://github.com/mattermost/mattermost-webapp/blob/2f0cc730068c208d53b0e91c2c885db6b4e02a23/types/cloud/sku.ts#L23-L37

Reply:
got it.